src/main/java/com/redhat/devtools/intellij/tektoncd/actions/logs/LogsBaseAction.java

```diff
@@ -44,12 +44,30 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
             Class<?> nodeClass = selected.getClass();
             String resourceName = pickRun(namespace, selected.toString(), nodeClass, anActionEvent.getPresentation().getText(), tkncli);
             if (resourceName == null) return;
+            String kind = getKind(selected, nodeClass);
 
-            this.actionPerformed(namespace, resourceName, nodeClass, tkncli);
+            this.actionPerformed(namespace, cleanName(resourceName), kind, nodeClass, tkncli);
         });
     }
 
-    public abstract void actionPerformed(String namespace, String resourceName, Class nodeClass, Tkn tkncli);
+    public abstract void actionPerformed(String namespace, String resourceName, String kind, Class nodeClass, Tkn tkncli);
+
+    private String cleanName(String resourceName) {
```

Comment:
No your are using an army: the node should have a name and if the visual representation is different then either you overload the toString method or you update the cell renderer.